ta/mk/build-user-ta.mk

```diff
@@ -54,3 +54,15 @@ include mk/compile.mk
 # Install TA libraries before in-tree TAs can be linked
 additional-link-deps := $(ta_dev_kit-files-lib)
 include  ta/arch/$(ARCH)/link.mk
+
+.PHONY: export_in_tree_tas
+export_in_tree_tas: $(out-dir)/export-$(ta-target)/ta/$(user-ta-uuid).ta
```

Comment:
How about letting `ta_dev_kit` depend on this `.ta` file instead? In a way we're updating the dev-kit when installing the TA file so it would make sense to let all files be copied first before considering the `ta_dev_kit` target complete.

Reply:
Good idea, in fact I wrongly assumed we had a circular dependency (with `.ta` depending on `ta_dev_kit`) but fortunately it is not the case. Updated.